### PR TITLE
Militia offroads & civilian CH-49 Retextures

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Adds retextured FIA offroads to the vanilla factions - color choice depending on environment or what would be faction-suitable.
Adds the Ch-49 in its civilian livery as a purchasable helicopter, for 6500€.

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Militia offroads: Load template, observe militia roadblocks
Civilian helicopter: buy the helicopter

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 1]
```diff
@@ -0,0 +1,123 @@
+class B_G_Offroad_01_armed_F;
+class B_G_Offroad_01_F;
+class B_G_Offroad_01_AT_F;
+class I_E_Offroad_01_comms_F;
+class I_E_Offroad_01_covered_F;
+//Black - Any/ION
+class a3a_Offroad_01_black_F	:	B_G_Offroad_01_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:7]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+{
+	scope = 2;
+	hiddenSelectionsTextures[] = {"a3\soft_f_enoch\offroad_01\data\offroad_01_ext_blk_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_ext_blk_co.paa"};
+	textureList[] = {};
+	animationList[] = {"HideDoor1",0,"HideDoor2",0,"HideDoor3",0.333333,"HideBumper1",0.5,"HideBumper2",0.5};
+};
+class a3a_Offroad_01_black_armed_F	:	B_G_Offroad_01_armed_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:14]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+};
+class a3a_Offroad_01_black_AT_F : B_G_Offroad_01_AT_F
+{
+	scope = 2;
+	hiddenSelectionsTextures[] = {"a3\soft_f_enoch\offroad_01\data\offroad_01_ext_blk_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_ext_blk_co.paa"};
+	textureList[] = {};
+	animationList[] = {"HideDoor1",1,"HideDoor2",1,"HideDoor3",1,"HideBumper1",0.5,"HideBumper2",0.5};
+};
+//Greens - Tropical NATO
+class a3a_Offroad_01_green_F	:	B_G_Offroad_01_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:29]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+{
+	scope = 2;
+	hiddenSelectionsTextures[] = {"a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa"};
+	textureList[] = {"Green",1};
+};
+class a3a_Offroad_01_green_armed_F	:	B_G_Offroad_01_armed_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:35]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+	hiddenSelectionsTextures[] = {"a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa"};
+	textureList[] = {"Green",1};
+};
+class a3a_Offroad_01_green_AT_F : B_G_Offroad_01_AT_F
+{
+	scope = 2;
+	hiddenSelectionsTextures[] = {"a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa"};
+	textureList[] = {"Green",1};
+};
+class a3a_Offroad_01_comms_green_F	:	I_E_Offroad_01_comms_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:47]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+	textureList[] = {"Green",1};
+};
+class a3a_Offroad_01_covered_green_F : I_E_Offroad_01_covered_F
+{
+	scope = 2;
+	hiddenSelectionsTextures[] = {"a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_ext_grn_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_cover_grn_co.paa"};
+	textureList[] = {"Green",1};
+};
+//Tans - Arid NATO
+class a3a_Offroad_01_tan_F	:	B_G_Offroad_01_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:60]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+{
+	scope = 2;
+	textureList[] = {"Beige",1};
+	hiddenSelectionsTextures[] = {"a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa","a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa"};
+};
+class a3a_Offroad_01_tan_armed_F	:	B_G_Offroad_01_armed_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:66]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+	textureList[] = {"Beige",1};
+	hiddenSelectionsTextures[] = {"a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa","a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa"};
+};
+class a3a_Offroad_01_tan_AT_F : B_G_Offroad_01_AT_F
+{
+	scope = 2;
+	textureList[] = {"Beige",1};
+	hiddenSelectionsTextures[] = {"a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa","a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa"};
+};
+class a3a_Offroad_01_comms_tan_F	:	I_E_Offroad_01_comms_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:78]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+	hiddenSelectionsTextures[] = {"a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa","a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_cover_blk_co.paa"};
+};
+class a3a_Offroad_01_covered_tan_F : I_E_Offroad_01_covered_F
+{
+	scope = 2;
+	textureList[] = {"Beige",1};
+	hiddenSelectionsTextures[] = {"a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa","a3\soft_f\offroad_01\data\offroad_01_ext_base01_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_cover_blk_co.paa"};
+};
+//LDF Camo
+class a3a_Offroad_01_ldf_armed_F	:	B_G_Offroad_01_armed_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:91]: change tabs to space

File: A3A/addons/config_fixes/Vanilla/soft.hpp [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+class a3a_Offroad_01_ldf_AT_F : B_G_Offroad_01_AT_F
+{
+	crew = "I_E_Soldier_F";
+	faction = "IND_E_F";
+	side = 2;
+	hiddenSelectionsTextures[] = {"a3\soft_f_enoch\offroad_01\data\offroad_01_ext_eaf_co.paa","a3\soft_f_enoch\offroad_01\data\offroad_01_ext_eaf_co.paa"};
+	textureList[] = {"EAF",1};
+};
+//Gendarmerie
+class a3a_Offroad_01_GEN_armed_F	:	B_G_Offroad_01_armed_F
```
Comment (Bob-Murphy) [inline A3A/addons/config_fixes/Vanilla/soft.hpp:108]: change tabs to space

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

**Comment by ante185** [comment]
Tabs exchanged for spaces, merge conflicts solved.

**Review by Bob-Murphy — APPROVED** [review]

**Comment by jaj22** [comment]
This should be on ready-for-merge now, right?

**Review by Bob-Murphy — APPROVED** [review]
lgtm

**Pull request merged**